.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 tota11y/
+build/
```

Comment:
Given our current build/release process, I'm not sure we can ignore the `build` directory just yet.

Reply:
Yeah, I see in your master that the build folder is included in the repo. I'll revert this.

Reply:
Done